### ISSUE TITLE
🎨 Share what you have made!

### ISSUE THREAD

**Issue by alexreardon** [atlassian/pragmatic-drag-and-drop#6]
We would love to see what you create with Pragmatic drag and drop. Please feel free to post a screen recording of your experience in this issue ✨

**Comment by glani** [comment]
when can we expect the source code ?

**Comment by jeffdaley** [comment]
I'll go. Really liking the library so far!

https://github.com/atlassian/pragmatic-drag-and-drop/assets/754957/e052c4be-8de9-4038-ad31-745b5662c407

Built this with 🐹 Ember.js despite zero understanding of React 🙂. Worked great with existing animation logic.

Source code is open source ([class](https://github.com/hashicorp-forge/hermes/blob/main/web/app/components/project/resource-list-item.ts) | [template](https://github.com/hashicorp-forge/hermes/blob/main/web/app/components/project/resource-list-item.hbs)) if anyone wants to refactor it for me, lol.

**Comment by clearfeld** [comment]
I'll go as well, added resizing to our data preview area.

![resizeable-preview-area](https://github.com/atlassian/pragmatic-drag-and-drop/assets/49600278/1503719b-d212-4595-bb97-ef1ce47a9620)

It was super smooth adding this in 🙌. I'm sure my team and I are going to be making extensive use of this library across the entire site.

Also if anyone wants to try using our mock data generator we have a playground here that's free to use (no signup needed).
https://www.dataconstruct.io/organizations/playground/schemas

**Comment by cdaz5** [comment]
I have a use case for multi drag (can select several items in a list). I slapped together a POC from the list example that was provided in the docs (hold shift to select multiple). @alexreardon would love to get your thoughts on if there is a api through the library I should be leveraging or this would be the recommended approach? Also would be happy to clean this up if you guys wanted to add it to the examples in the docs.

basically the TLDR is have additional state that tracks what has been "selected" by clicking.

https://codesandbox.io/p/sandbox/dnd-multi-drag-example-jcymvd?file=%2Fexample.tsx%3A71%2C33

**Comment by alexreardon** [comment]
We will likely provide a multi-drag pattern similar to this: https://github.com/atlassian/react-beautiful-dnd/blob/master/docs/patterns/multi-drag.md

**Comment by alexreardon** [comment]
Let's move this conversation off of an issue and into a discussion :) 

https://github.com/atlassian/pragmatic-drag-and-drop/discussions/categories/show-and-tell
